Twig/CookieGuardExtension.php

```diff
@@ -62,6 +63,11 @@ public function showIfCookieAccepted($html)
         ]);
     }
 
+    public function cookieSettingsAreAccepted()
```

Comment:
Deze method dan ook gebruiken ipv de bestaande check op regel 58, dan houden we de logica op 1 centraal punt

Reply:
Done